metrics_utility/test/gather/test_directory_gather.py

```diff
@@ -0,0 +1,47 @@
+import glob
+import os
+import pytest
+import subprocess
+import sys
+from datetime import datetime
+
+env_vars = {
+    'METRICS_UTILITY_REPORT_TYPE': 'CCSPv2',
+    'METRICS_UTILITY_SHIP_PATH': './metrics_utility/test/test_data',
+    'METRICS_UTILITY_SHIP_TARGET': 'directory',
+    'AWX_LOGGING_MODE': 'stdout',
+}
+
+year = datetime.now().strftime('%Y')
+uuid = '00000000-0000-0000-0000-000000000000'  # mock_awx INSTALL_UUID setting
+
+file_glob = f'./metrics_utility/test/test_data/data/{year}/*/*/{uuid}-*.tar.gz'
+
+
+def validate_exists(file_glob):
+    assert len(glob.glob(file_glob)) > 0
+
+
+@pytest.fixture
+def cleanup_glob():
```

Comment:
When does this get called?

Reply:
It gets called around the `test_command` test (because of the `cleanup_glob` param),
so the bits after the `yield` get called after the test has run, anything before the yield would get called before.